Build/GetWix.ps1

```diff
@@ -0,0 +1,21 @@
+function GetWix {

```

Comment:
Is there a reason this is more complicated than the following?
https://github.com/rackerlabs/dotnet-collections/blob/master/build/build.ps1#L32-L39